src/main/java/de/retest/web/WebDataFilter.java

```diff
@@ -0,0 +1,28 @@
+package de.retest.web;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+// TODO This should be part of recheck and somehow use RecheckAdapter#convert to filter the result.
+public class WebDataFilter {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=94&issues=AWbuNaawTJjFgJLTqtx7&open=AWbuNaawTJjFgJLTqtx7)

[ISSUE_KEY]: AWbuNaawTJjFgJLTqtx7